format-all.el

```diff
@@ -853,6 +854,12 @@ Consult the existing formatters for examples of BODY."
   (:languages "_Snakemake")
   (:format (format-all--buffer-easy executable "-")))
 
+(define-format-all-formatter zig
+  (:executable "zig")
+  (:install)
+  (:languages "_Zig")
```

Comment:
Since Zig has now been added to [language-id](https://github.com/lassik/emacs-language-id), this can be `(:languages "Zig")` without the underscore.